Justfile

```diff
@@ -74,9 +74,9 @@ all-release: build-release test-release
 # build the man pages
 @man:
     mkdir -p "${CARGO_TARGET_DIR:-target}/man"
-    version=$(cat Cargo.toml | grep ^version | head -n 1 | awk '{print $NF}' | tr -d '"'); \
+    version=$(awk 'BEGIN { FS = "\"" } ; /^version/ { print $2 ; exit }' Cargo.toml); \
```

Comment:
Interesting, always good to learn some `awk` magic. ... Definitely makes this more elgant, but not sure it's more readable though, hmm.

Reply:
Yeah, i wasn't sure that putting something too tricky was a good idea… But as that's the only call of `awk` command I replace everything by one "big" awk command.
I also think to something more readable : 
`awk '/^version/ { print $NF ; exit }' Cargo.toml | tr -d '"'`

But you can also reduce the elements with direct call to `grep` with the right option : 
`grep "^version" --max-count=1 Cargo.toml |  awk '{print $NF}' | tr -d '"'`

Reply:
No I think it's fine with just awk ... at least no one will ever attempt to change that ever again :D